.gitignore

```diff
@@ -33,4 +33,5 @@ coverage.txt
 # Emacs save files have literal pound signs/octothorpes/hashtags in them
 # So the \# escapes # as comment syntax
 \#*#
-.\#*
\ No newline at end of file
+.\#*
+sagalog/*
```

Comment:
recommend update the test to remove the directory (on success)... eliminating this ignore.